Blazor/wwwroot/cdts-blazor.js

```diff
@@ -0,0 +1,296 @@
+﻿//Get and set the culture name in the local storage
+window.blazorCulture = {
+    get: () => window.localStorage['BlazorCulture'],
+    set: (value) => window.localStorage['BlazorCulture'] = value
+};
+
+//Define global variables
+var exitScriptObj = null;
```

Comment:
(Bascially applies to all variables and functions defined here)

To avoid poluting the global namespace, all of these should be in an `cdtsBlazor` object.

```javascript
var cdtsBlazor = {
   exitScriptObj: null,
   ...

  setRefTop = function setRefTop() {
...
  }
..
};
```

Reply:
done

Reply:
done